rancher/v1.2/en/cattle/adding-load-balancers/index.md

```diff
@@ -285,206 +322,201 @@ Rancher supports wildcards when adding host based routing. The following wildcar
+
+>**Note:** For any load balancers using the v1 load balancer yaml fields that uses selector labels will not be converted to the v2 load balancer as the port rules on the service would not be updated.
+
+###### Example `docker-compose.yml`
 
 ```yaml
-lb:
-  image: rancher/load-balancer-service
-  ports:
-  # Listen on public port 80 and direct traffic to private port 80 of the service
-  - 80:80
-  # Listen on public port 82 and by default forward traffic to private port 81 using HTTP
-  - 82:81
-  # Listen on public port 9999 using TCP and not HTTP
-  - 9999/tcp
-  links:
-  # Target services in the same stack will be listed as a link
-  - web1:web1a
-  - web2:web2a
-  external_links:
-  # Target services in a different stack will be listed as an external link
-  - differentstack/web3:web3a
-  labels:
-    # Put load balancer containers on hosts with label lb=true
-    io.rancher.scheduler.affinity:host_label: lb=true
-    # Requests to http://app.example.com/foo:80 should be routed to web1 over port 8000
-    io.rancher.loadbalancer.target.web1: app.example.com:80/foo=8000
-    # Requests to http://app.example.com/foo should be routed to web2 over port 8000
-    # and http://app.example.com/foo/bar over port 8001
-    io.rancher.loadbalancer.target.web2: app.example.com/foo=8000,app.example.com/foo/bar=8001
-    # Requests routed to web3 go to port 8000, overriding the default configuration
-    # of 82:81
-    io.rancher.loadbalancer.target.differentstack/web3: 82=8000
+version: '2'
+services:
+  lb1:
+    image: rancher/lb-service-haproxy
+    ports:
+    - 81
+  web:
+    image: nginx
+    # This service will be picked up by the load balancer as a target as
+    # the selector in the load balancer would pick up this service
+    labels:
+      foo: bar
 ```
 
-##### Example `rancher-compose.yml`
+###### Example `rancher-compose.yml`
 
 ```yaml
-lb:
-  # Two load balancer containers
-  scale: 2
-  load_balancer_config:
-    name: lb config
-  health_check:
-    port: 42
-    interval: 2000
-    unhealthy_threshold: 3
-    healthy_threshold: 2
-    response_timeout: 2000
-web1:
-  scale: 1
-web2:
-  scale: 1
-web3:
-  scale: 1
+version: '2'
+services:
+  lb1:
+    scale: 1
+    lb_config:
+      port_rules:
+      - source_port: 81
+        # Target any service that has foo=bar as a label
+        selector: foo=bar
+        protocol: http
+    health_check:
+      port: 42
+      interval: 2000
+      unhealthy_threshold: 3
+      healthy_threshold: 2
+      response_timeout: 2000
+  web:
+    scale: 1
+    lb_config:
+      port_rules:
+      - target_port: 80
+        hostname: test.com    
 ```
 
-##### Examples of Using Multiple Ports in Advanced Routing Options
+##### Backend Name
 
-Regardless of the priority, every service specified on the load balancer is registered for every load balancer listening port.
+If you want to explicitly label your backend in your load balancer configuration, you would use the `backend_name`.
```

Comment:
we should mention that this option can be useful when you want to configure custom config parameters for a particular backend.